content/docs-v1/collection-catalogues.md

```diff
@@ -1,126 +1,323 @@
+## Creating an `objects.yaml` File
+
+This first step of creating a collection catalogue in Quire is to capture all the data related to your various objects in an  `objects.yaml` file. Similar to how [*Figure Images*](/docs-v1/figure-images/) are managed with YAML, this data can then be pulled into your Markdown files through the use of an `id`. The primary way this information gets displayed is as a table of information that accompanies each object in the publication. This table is often referred to as a "tombstone". 
+
+{{< q-figure id= "tombstone" >}}
+
+There are two steps required when setting up your `objects.yaml` file. The first step is to create an  `object_display_order` list. This controls not only what information will appear in the tombstone, but also the order in which it appears.  
 
 ```yaml
 object_display_order:
   - artist
+  - title
   - year
   - dimensions
   - medium
   - location
+```
+
+Once you have determined the information you want to display, you then create an `object_list`. Similar to entries in a `figures.yaml` file, each entry in this list will reference a different object featured in your publication. The attributes that populate this list should include, but are not limited to the attributes listed under `object_display_order`. For example, you may want categorize types of objects (painting, sculpture, drawing) for use in the [filterable object grid](#generate-object-grids) but do not require this information to show up in the object tombstone. In that case you would leave `type` out of `object_display_order` but keep it in your `object_list`. 
+
+```yaml
 object_list:
-  - id: "2"
-    title: "Impression, *Sunrise*"
-    artist: "Claude Monet"
-    year: "1872"
-    medium: "Oil on canvas"
-    dimensions: "48 cm × 63 cm (18.9 in × 24.8 in)"
-    location: "Musée Marmottan Monet, Paris"
-    link:
-    figure:
-      - id: "cat2"
-  - id: "3"
-    title: "Reading (portrait of Edma Morisot)"
-    artist: "Berthe Morisot"
-    year: "1873"
-    medium: "Oil on fabric"
-    dimensions: "74.2 x 100.3 x 12 cm (29 3/16 x 39 1/2 x 4 11/16 in.)"
-    location: "Cleveland Museum of Art"
-    link: http://www.clevelandart.org/art/1950.89
-    download: true
+  - id: cat-1
+    artist:
+    title:
+    year:
+    dimensions:
+    medium:
+    location:
+    type:
+    link: 
     figure:
-      - id: "cat3"
-      - id: "cat3a"
-      - id: "cat3b"
+      - id: 
+  - id: cat-2
+    artist:
+    title: 
+    year:
+    dimensions:
+    medium:
+    location:
+    type:
+    link: 
+    figure: 
+      - id: 
 ```
 
-There are two sections in the `objects.yaml` file:  `object_display_order` and `object_list`.
+The attributes you choose to include for your objects are completely up to you. However, there are some attribute names that are reserved for particular uses. They are `id`, `figure`, `link`, and `thumbnail`. These should be included in the `object_list` but do not need to be included in the `object_display_order`. 
```

Comment:
I'd cut the last sentence (which felt not 100% correct to me) and add "as detailed below" to the end of the new last sentence.